packages/fuel-indexer-lib/src/defaults.rs

```diff
@@ -14,6 +14,7 @@ pub const POSTGRES_PASSWORD: &str = "postgres";
 pub const INDEX_FAILED_CALLS: usize = 10;
 
 pub const GRAPHQL_API_RUN_MIGRATIONS: bool = false;
+pub const MAX_BODY_LIMIT: &str = "10"; // 5MB in bytes
```

Comment:
- So I see [`new`](https://docs.rs/tower-http/latest/tower_http/limit/struct.RequestBodyLimitLayer.html#method.new) takes a `usize` but here we're making `MAX_BODY_LIMIT` a `String`. 
- I'm assuming `clap` doesn't allow `usize` values in command fields?
- I also see we're assigning it to a value of `"10"` ? I would expect the value (even if a string) to be what you added in `fuel-indexer-test/lib/defaults` i.e., `pub const MAX_BODY_LIMIT: usize = 5242880;`

Reply:
My bad, left this in for testing. Will fix. I can attempt to coeerce clap with `usize`, I'm sure its possible, that will be cleaner yes.

Reply:
So clap does allow `usize` type in command fields, but it requires the type `&str` to set the `default_value` in the `#[clap()]` attribute. For that reason I've left it as a string, with the conversion to `usize` happening on 302. Let me know if you prefer a different way.